partials/homescreen.html

```diff
@@ -11,21 +11,21 @@
 		</div>
 	</div>
 
-	<div ng-if="user.mentee === 1">
+	<div ng-if="user.admin">
```

Comment:
For some reason this div's height does not contain the toggle requesting period widget, which means that the "Your Mentor" section is creeping up over the widget and partially covers the "open widget" button. Not sure what the fix is--probably some nasty CSS bug =(